include/crocoddyl/core/utils/file-io.hpp

```diff
@@ -0,0 +1,85 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020 LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+// modified from https://gist.github.com/rudolfovich/f250900f1a833e715260a66c87369d15
+
+#ifndef CROCODDYL_CORE_UTILS_FILE_IO_HPP_
+#define CROCODDYL_CORE_UTILS_FILE_IO_HPP_
+
+#include <string>
+#include <fstream>
+#include <sstream>
+
+class CsvStream {
+  std::ofstream fs_;
+  bool is_first_;
+  const std::string separator_;
+  const std::string escape_seq_;
+  const std::string special_chars_;
+
+ public:
+  CsvStream(const std::string filename, const std::string separator = ",")
+      : fs_(), is_first_(true), separator_(separator), escape_seq_("\""), special_chars_("\"") {
+    fs_.exceptions(std::ios::failbit | std::ios::badbit);
+    fs_.open(filename);
+  }
+
+  ~CsvStream() {
+    flush();
+    fs_.close();
+  }
+
+  void flush() { fs_.flush(); }
+
+  inline static CsvStream& endl(CsvStream& file) {
+    file.endrow();
+    return file;
+  }
+
+  void endrow() {
+    fs_ << std::endl;
+    is_first_ = true;
+  }
+
+  CsvStream& operator<<(CsvStream& (*val)(CsvStream&)) { return val(*this); }
```

Comment:
It seems to me this file doesn't print an output in the terminal.
I believe it would be good to do it as well.

I know you have prepared a Python script for nice processing, this is good. However, I still think that having an extra output in the terminal is useful.

Reply:
See my comment above.